perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -4480,6 +4530,27 @@ private void processNotificationsAfterCreation(PerunSession session, Application
 			}
 		}
 	}
+
+	/**
+	 * Returns ResultSetExtractor that can be used to extract returned paginated applications
+	 * from db.
+	 *
+	 * @param query query data
+	 * @return extractor, that can be used to extract returned paginated applications from db
+	 */
+	private static ResultSetExtractor<Paginated<Application>> getPaginatedApplicationsExtractor(ApplicationsPageQuery query) {
```

Comment:
Nit: you can move this method under `---MAPPERS AND SELECTS---` below because it is related logic.